Meadowlark-js/backends/meadowlark-aws-dynamodb-opensearch/src/index.ts

```diff
@@ -3,23 +3,24 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
-import { MeadowlarkBackendPlugin } from '@edfi/meadowlark-core';
+import { DocumentStorePlugin, QueryHandlerPlugin } from '@edfi/meadowlark-core';
```

Comment:
Split backend plugin interface into two. One for document store and reference validation (create, update, delete, get by id) and one for handling querys. Can be the same plugin in practice.